README.md

```diff
@@ -68,27 +65,32 @@ WORKFLOW_INSTANCE_ID= #put id here
 
 ### Start a workflow
 
-Start and the workflow
+Start the workflow
 
 ``` bash
 ./ensemble.sh instances start $WORKFLOW_INSTANCE_ID
 ```
 
 ## More Workflow Examples
 
+``` bash
 ./ensemble.sh workflows create ./samples/dca.workflow.yaml
-WORKFLOW_ID=
```

Comment:
this meant:
WORKFLOW_ID= #put here the received workflow id
I'd like to put this back as it was convenient for me. wdyt?

Reply:
Yes @leonprou  now with these details it makes more sense to keep this line but to add also the comment to explain as in other parts.